nl_server/embeddings_map.py

```diff
@@ -157,3 +160,7 @@ def _merge_custom_index(default: EmbeddingsIndex,
       f'custom {custom.embeddings_local_path} into {output_embeddings_path}')
 
   return output_embeddings_path
+
+
+def _is_custom_dc() -> bool:
```

Comment:
Can we add this method to the shared lib so it can be used in other places?

Reply:
There's not a good file, and need to create one for this, lets move it when its used another time?

Reply:
SG